### DISCUSSION TITLE
nosign ipa sidestore/altstore 自签 ldid 报错

### DISCUSSION THREAD

**Discussion by cloud-beta-0** [xiaohucode/yidaRule#250] [General]
xx-release-nosign.ipa 包在sidestore/altstore中自签得到 ldid 错误

side store任意版本
ipad pro 10.5 - 17.0.3

`/Users/runner/Library/Developer/Xcode/DerivedData/AltStore-dunvejkuqerofjghtqolpzgexuyv/SourcePackages/checkouts/AltSign/Sources/ldid/../ldid-core/ldid.cpp(1368): _assert(): end >= size - 0x10`

尝试了1.1.15-release.ipa; 1.1.14+2-release.ipa正常安装

**新版本可以继续发布签名包吗**

**Comment by kezhihao** [comment]
我也用的altstore安装失败，用sideloadly安装就可以了，亲测有效

**Reply by cloud-beta-0** [reply]
Sideloadly 是可以就是重签有点麻烦
